Gopkg.toml

```diff
@@ -35,7 +35,7 @@ required = [
 # Our master branch tracks knative/pkg master or a release.
 [[override]]
   name = "knative.dev/pkg"
-  branch = "release-0.9"
```

Comment:
please don't revert this. We need to be on the 0.9 branch for the Tuesday release.

/hold

Reply:
when was pkg 0.9 done? So all changes that were included between then and tuesday won't be able to get in the release? Why can't we just release with a version of master for example?

Reply:
I followed https://github.com/knative/pkg/blob/master/RELEASING.md mostly on timing. This friday is the day we are suppose to have all projects on the pkg release.

Reply:
you can still update pkg, and make a PR for the 0.9 branch (i had to do this once already for serving) and then a followup PR for all projects for the new revision of the pkg repo.

Reply:
I see, OK, well will see on monday if it's worth it. 
Let's keep the hold on this then